pulp.spec

```diff
@@ -325,6 +325,7 @@ touch %{buildroot}/%{_sysconfdir}/pki/%{name}/consumer/server/rsa_pub.key
 cp -R agent/etc/pulp/agent/agent.conf %{buildroot}/%{_sysconfdir}/%{name}/agent/
 cp -R client_consumer/etc/pulp/consumer/consumer.conf %{buildroot}/%{_sysconfdir}/%{name}/consumer/
 %if 0%{?rhel} >= 6 || 0%{?fedora} >= 19
+mkdir -p %{buildroot}/%{_sysconfdir}/bash_completion.d
```

Comment:
As an aside, the `install` executable is extremely useful for this sort of thing.

```console
$ tree
.

0 directories, 0 files
$ touch file
$ touch executable
$ install -Dm644 file foo/bar/file  # compact syntax
$ install -D -m 755 executable foo/biz/executable
$ tree
.
├── executable
├── file
└── foo
    ├── bar
    │   └── file
    └── biz
        └── executable

3 directories, 4 files
$ ls -lha foo/*/*
-rw-r--r-- 1 ichimonji10 ichimonji10 0 Feb  8 15:17 foo/bar/file
-rwxr-xr-x 1 ichimonji10 ichimonji10 0 Feb  8 15:18 foo/biz/executable
```